### PR TITLE
[Main][Task]27156360: Add json config cdn details to tool folder

### PR DESCRIPTION
**Pull request by Karlie-777**

### CODE REVIEW

File: tools/cfgsync-cdn/README.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+| featureOptIn.iKeyUsage.offCfg | override values of the following cdn `config` when feature-iKeyUsage is disabled | <sub>DefaultThrottleMsgKey</sub></br>throttleMgrCfg.109.enabled: `false`</br> <sub>InstrumentationKeyDeprecation</sub></br> throttleMgrCfg.106.enabled: `false`
+| config | override values for user's core config | throttleMgrCfg
+| config.throttleMgrCfg | override values for user's throttleMgrCfg under core config | <sub>InstrumentationKeyDeprecation</sub></br> throttleMgrCfg.106:</br>{  `"enabled": true`, *// will send ikey InstrumentationKey Deprecation message*</br>`"limit": { "samplingRate": 1, "maxSendNumber": 1}`, *// sampling rate: 0.0001%, and will send max one message per time* </br>`"interval": {"monthInterval": 6,"daysOfMonth": [1]}`} *// message will be sent on the first day every 6 month*,</br><sub>DefaultThrottleMsgKey</sub></br>throttleMgrCfg.109:</br>{  `"enabled": false`, *// will not send default ikey message*</br> ... *// all other settings are same with the InstrumentationKeyDeprecation settings*}</br>
+
+## Contributing
```
Comment (Karlie-777) [inline tools/cfgsync-cdn/README.md:21]: every 2month, 1st day

File: tools/cfgsync-cdn/README.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+
```
Comment (Karlie-777) [inline tools/cfgsync-cdn/README.md:22]: create a task: change the beta endpoint to the doc

File: tools/cfgsync-cdn/ai.cfgsync.json [hunk 3]
```diff
@@ -0,0 +1,53 @@
+{
+    "name": "Application Insights CfgSync CDN",
+    "description":"Application Insights throttleMgrCfg Configurations and feature opt-in Details",
+    "version": "1.0.0",
+    "homepage": "https://github.com/microsoft/ApplicationInsights-JS#readme",
+    "author": "Microsoft Application Insights Team",
+    "sideEffects": false,
+    "repository": {
+        "type": "git",
+        "url": "https://github.com/microsoft/ApplicationInsights-JS"
```
Comment (Karlie-777) [inline tools/cfgsync-cdn/ai.cfgsync.json:10]: do not need any "normal setting", no MIT needed as well

File: tools/cfgsync-cdn/ai.cfgsync.json [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+    },
+    "enabled": true,
```
Comment (Karlie-777) [inline tools/cfgsync-cdn/ai.cfgsync.json:12]: add scripts folder, add copy task,

Comment (Karlie-777) [reply tools/cfgsync-cdn/ai.cfgsync.json:12]: and add version number after the json name

Comment (Karlie-777) [reply tools/cfgsync-cdn/ai.cfgsync.json]: actually, I created the config folder a long time ago with build task defined and release scripts. so moved the CDN to that folder and changed the version from 0.1.0 to 1.0.0

File: tools/config/package.json [hunk 5]
```diff
@@ -1,7 +1,7 @@
 {
     "name": "applicationinsights-web-config",
-    "description": " Application Insights JavaScript SDK - Web Config",
```
Comment (Karlie-777) [inline tools/config/package.json:3]: actually, I created the config folder a long time ago with build task defined and release scripts. so moved the CDN to that folder and changed the version from 0.1.0 to 1.0.0

File: tools/cfgsync-cdn/README.md (hunk 2 above)
Comment (Karlie-777) [reply tools/cfgsync-cdn/README.md]: Task 27176994 is created

**Review by MSNev — APPROVED** [review]

**Pull request merged**